packages/app/src/AuthPage.tsx

```diff
@@ -153,15 +174,22 @@ export default function AuthPage({ action }: { action: 'register' | 'login' }) {
                       htmlFor="confirmPassword"
                       className="text-start text-muted fs-7.5 mb-1"
                     >
-                      Confirm Password
+                      <CheckOrX
+                        handler={confirmPass}
+                        password={currentPassword}
+                      >
+                        Confirm Password
+                      </CheckOrX>
                     </Form.Label>
                     <Form.Control
                       data-test-id="form-confirm-password"
                       id="confirmPassword"
                       type="password"
                       className="border-0"
```

Comment:
```suggestion
                      className="border-0 mb-2"
```
just adds a bit of breathing room between the input

<img width="510" alt="image" src="https://github.com/hyperdxio/hyperdx/assets/2781687/34288d16-be85-4222-8c21-b7fa54761dfc">
vs
<img width="454" alt="image" src="https://github.com/hyperdxio/hyperdx/assets/2781687/554bb8ea-1327-4527-b6cb-fcecb1982fcf">